README.md

```diff
@@ -51,7 +51,7 @@ Job and data sources configurations allow you to flexibly adjust the import to y
 
 
 3. Data set - load files, source files (native documents, images, text files) - need to be placed in the destination fileshare location accessible to workspace. 
-
+   You can use TAPI (https://github.com/relativitydev/transfer-api-samples) to upload all files to destination location.
```

Comment:
this will forward user to a deprecated examples and library, I think we we need to use new Transfer.SDK examples if they exists or give them direct nuget: https://www.nuget.org/packages/Relativity.Transfer.SDK/

Reply:
Yes, I see :) I would rather place here some new TAPI documentation for user reference.

Reply:
updated link to new TAPI documentation